examples/backends/vllm/launch/disagg_omni_glm_image_nixl.sh

```
@@ -0,0 +1,168 @@
+#
+# Examples:
+#   AR on XPU TP2:  ZE_AFFINITY_MASK=2,3 ROLE=ar AR_TP=2 AR_DEVICE_TYPE=xpu
+#   AR on CUDA:     CUDA_VISIBLE_DEVICES=0 ROLE=ar AR_TP=1 AR_DEVICE_TYPE=cuda
+#   DiT on CUDA:    CUDA_VISIBLE_DEVICES=5 ROLE=dit DIT_TP=1 DIT_DEVICE_TYPE=cuda
+#   DiT on XPU:     ZE_AFFINITY_MASK=0 ROLE=dit DIT_TP=1 DIT_DEVICE_TYPE=xpu
+
+set -e
+trap dynamo_exit_trap EXIT
+
+SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
+source "$SCRIPT_DIR/../../../../common/launch_utils.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./../../../../common/launch_utils.sh was not specified as input (see shellcheck -x).

(SC1091)